array-manipulation-1/src/compact.js

```diff
@@ -0,0 +1,11 @@
+/* exported compact */
+
+function compact(array) {
+  var a = [];
+  array.forEach(b => {
```

Comment:
interesting approach! Nothing wrong with it as long as you know how all the parts are working. If you don't I recommend asking about it! I bring that up because you don't see a lot of arrow functions and forEach used at this stage.

Reply:
I was looking at the array methods page and keep looking for a  method that would help with what I was trying to do then my brother suggested looking at the .forEach method and finding out about falsy and truthy figured i would try and see what happened then took it apart piece by piece to understand how it works